productized/logic/build-quarkus-app.sh

```diff
@@ -44,11 +44,11 @@ properties_with_versions=("compiler-plugin.version:3.11.0" "surefire-plugin.vers
 # extra dependencies required for offline mode
 quarkus_extensions_extra_deps="com.aayushatharva.brotli4j:native-linux-aarch64:1.12.0"
 # common extensions used by the kogito-swf-builder and kogito-swf-devmode
-quarkus_extensions="quarkus-kubernetes,smallrye-health,org.kie.kogito:kogito-quarkus-serverless-workflow:${kogito_version},org.kie.kogito:kogito-addons-quarkus-knative-eventing:${kogito_version},org.kie.kogito:kogito-addons-quarkus-microprofile-config-service-catalog:${kogito_version},org.kie.kogito:kogito-addons-quarkus-kubernetes:${kogito_version},org.kie.kogito:kogito-addons-quarkus-knative-serving:${kogito_version}"
+quarkus_extensions="quarkus-kubernetes,smallrye-health,org.kie.kogito:kogito-quarkus-serverless-workflow:${kogito_version},org.kie.kogito:kogito-addons-quarkus-knative-eventing:${kogito_version},org.kie.kogito:kogito-addons-quarkus-microprofile-config-service-catalog:${kogito_version},org.kie.kogito:kogito-addons-quarkus-kubernetes:${kogito_version},org.kie.kogito:kogito-addons-quarkus-knative-serving:${kogito_version},org.kie.kogito:kogito-addons-quarkus-jobs-knative-eventing:${kogito_version},org.kie.kogito:kogito-addons-persistence-jdbc:${kogito_version}"
```

Comment:
By adding the kogito-addons-persistence-jdbc to the swf-builder directly, you are forcing that all workflows produced by the swf-builder must have persistence. And if not, the produced build will never start, which means that it's not possible to have workflows without persistence. Also, If I'm not wrong, unlike adding an extension at builder, when an extension was added into the builder in this way, it's not possible to produce a build by not using that extension. So, basically we are saying that every build produced by the builder must have persistence.
Is this behavior what we want?

@rgdoliveira @ricardozanini  @jakubschwan ?